pkg/controller/import-controller.go

```diff
@@ -555,6 +584,26 @@ func (r *ImportReconciler) createScratchPvcForPod(pvc *corev1.PersistentVolumeCl
 	return nil
 }
 
+// Get path to VDDK image from 'v2v-vmware' ConfigMap
+func (r *ImportReconciler) getVddkImageName() (*string, error) {
+	namespace := util.GetNamespace()
+
+	cm := &corev1.ConfigMap{}
+	err := r.uncachedClient.Get(context.TODO(), types.NamespacedName{Name: common.VddkConfigMap, Namespace: namespace}, cm)
```

Comment:
Any particular reason you are using the uncached client here? This will hit the api server each time to get the information and I don't think it will change much, so the regular client which caches should suffice.

Reply:
The only reason is that it's the only way I could make it work. With the regular client, I get:

`Failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:cdi:cdi-sa" cannot list resource "configmaps" in API group "" at the cluster scope`

I can look deeper into this if it's important to use the cached client instead.

Reply:
@mansam does it ring a bell? You fixed an issue with permissions in https://github.com/kubevirt/vm-import-operator/pull/388.

Reply:
It looks like the cdi-sa account's cluster role only gives it the "get" permission on configmaps. "list" would need to be added to the policy, probably here: https://github.com/kubevirt/containerized-data-importer/blob/29d3a5e393aa1bd78e8dd0c4e47d80a78c70d434/pkg/operator/resources/cluster/controller.go#L120

Reply:
The cached client likely will also need watch. But giving the sa cluster scoped watch and list on config maps might not be a good plan. Discussion this with the team. If we don't want to give it those permissions, then using the uncached client is fine.